velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -155,6 +162,65 @@ TEST_F(StringTest, sha1) {
       "a26704c04fc5f10db5aab58468035531cc542485");
 }
 
+TEST_F(StringTest, sha2) {
+  EXPECT_EQ(sha2(std::nullopt, 256), std::nullopt);
```

Comment:
Test cases when one of the arguments is null are redundant. This is handled by the framework, not the function itself.